helper.c

```diff
@@ -256,6 +256,7 @@ static void start_host_interface(void)
 
     xpc_dictionary_set_bool(desc, vmnet_enable_tso_key, options.enable_tso);
     xpc_dictionary_set_bool(desc, vmnet_enable_checksum_offload_key, options.enable_checksum_offload);
+    xpc_dictionary_set_bool(desc, vmnet_enable_isolation_key, options.enable_isolation);
```

Comment:
There are no docs about this, but maybe it works in only in host mode. When host mode works we can test this again.